pytorch_lightning/trainer/trainer.py

```diff
@@ -126,7 +126,7 @@ def __init__(
         checkpoint_callback: bool = True,
         callbacks: Optional[Union[List[Callback], Callback]] = None,
         default_root_dir: Optional[str] = None,
-        gradient_clip_val: float = 0.0,
+        gradient_clip_val: Union[int, float] = 0.0,
```

Comment:
Why the added `int` type? Does this do anything different?

Reply:
nope.. just pytorch has it.. and we also do in clip_gradients 
https://pytorch.org/docs/stable/generated/torch.nn.utils.clip_grad_norm_.html